modules/launch-templates/locals.tf

```diff
@@ -22,7 +22,7 @@ locals {
     block_device_mappings = {
       device_name           = "/dev/xvda"
       volume_type           = "gp3" # The volume type. Can be standard, gp2, gp3, io1, io2, sc1 or st1 (Default: gp3).
-      volume_size           = "200"
+      volume_size           = "100"
```

Comment:
this could potentially have a significant impact to users - should we leave this as is for now?

Reply:
Did you mean to change this back to 200 so that existing customers wont be impacted?

Reply:
Updated